app/src/main/java/com/jalasoft/routesapp/util/ConverterFromPixelToDpUtil.kt

```diff
@@ -0,0 +1,12 @@
+package com.jalasoft.routesapp.util
+
+import android.content.Context
+import android.util.TypedValue
+
+object ConverterFromPixelToDpUtil {
```

Comment:
Maybe DisplayUtils or ScreenUtils could be more general and reusable. Still nice job